src/memberBundle.ts

```diff
@@ -0,0 +1,120 @@
+import { getModelForClass } from '@typegoose/typegoose';
+import { addDays } from 'date-fns';
+import {
+  issueMemberCertificate,
+  retrieveVeraDnssecChain,
+  selfIssueOrganisationCertificate,
+  serialiseMemberIdBundle,
+} from '@relaycorp/veraid';
+
+import type { ServiceOptions } from './serviceTypes.js';
+import { MemberPublicKeyModelSchema } from './models/MemberPublicKey.model.js';
+import { MemberModelSchema } from './models/Member.model.js';
+import { Kms } from './utilities/kms/Kms.js';
+import { OrgModelSchema } from './models/Org.model.js';
+import { derDeserialisePublicKey } from './utilities/webcrypto.js';
+import { Result } from './utilities/result.js';
+
+export async function generateMemberBundle(
+  publicKeyId: string,
+  options: ServiceOptions,
+): Promise<Result<ArrayBuffer, {
+  shouldRetry: boolean
+}>> {
```

Comment:
I don't really like the response. 
Having Result<ArrayBuffer, boolean> has readability problem when you call the function. So if it fails we get 
```
{
      didSucceed: false,
      reason:true
}
```
And it's really hard to decide what does `reason:true/false` mean

This way it's a bit more visible but still maybe we could create a new type

Reply:
How about rename `FailureResult.reason` to `context`? "Context" is more generic and works in all cases.

Reply:
That will require to rework almost the whole codebase since it's used a lot

Reply:
This rework will take place in a dedicated PR. #128